### PR TITLE
Code formatting and checks are moved to a single module.

### LINKED ISSUE #537: Add code formatting as a script
Code formatting need to be individually handled for black and isort, this can be combined into a single script.

### PR DESCRIPTION
**Pull request by devrimcavusoglu**
Closes #537.

### CODE REVIEW

**Comment by fcakyon** [comment]
@devrimcavusoglu great PR!

`scripts` folder should be ignored during packaging https://github.com/obss/sahi/blob/f104d1bfbd52611400032c5e71262ac3b280bb45/setup.py#L35

File: setup.cfg [hunk 1]
```diff
@@ -1,9 +1,10 @@
 [flake8]
-max-line-length = 119
-max-complexity = 18
-exclude =.git,__pycache__,docs/source/conf.py,build,dist
-ignore = I101,I201,F401,F403,S001,D100,D101,D102,D103,D104,D105,D106,D107,D200,D205,D400,W504,D202,E203,E501,E722,W503,B006
-inline-quotes = "
+max-line-length = 127
```
Comment (fcakyon) [inline setup.cfg:2]: 120 would be better I believe
```suggestion
max-line-length = 120
```

File: setup.cfg [hunk 2]
```diff
@@ -1,9 +1,10 @@
 [flake8]
-max-line-length = 119
-max-complexity = 18
-exclude =.git,__pycache__,docs/source/conf.py,build,dist
-ignore = I101,I201,F401,F403,S001,D100,D101,D102,D103,D104,D105,D106,D107,D200,D205,D400,W504,D202,E203,E501,E722,W503,B006
-inline-quotes = "
```
Comment (fcakyon) [inline setup.cfg:6]: Why do we remove this line?

Comment (devrimcavusoglu) [reply setup.cfg:6]: Restored `setup.cfg`.

**Review by fcakyon — APPROVED** [review]

**Pull request merged**